src/server/server.pro

```diff
@@ -92,16 +92,16 @@ CONFIG(dsme_dbus_if) {
 target.path = $$(DESTDIR)/usr/bin
 
 timedrc.path  = $$(DESTDIR)/etc
-timedrc.files = timed-qt5.rc
+timedrc.files = timed-qt$${QT_MAJOR_VERSION}.rc
```

Comment:
Let's drop "-qt5" from the rc-file as well. This will imply that you'd need to update/fix configuration_path of the timed.h as well.